flow/layers/display_list_layer.cc

```diff
@@ -109,24 +109,24 @@ void DisplayListLayer::Paint(PaintContext& context) const {
   FML_DCHECK(display_list_.skia_object());
   FML_DCHECK(needs_painting(context));
 
-  SkAutoCanvasRestore save(context.leaf_nodes_canvas, true);
-  context.leaf_nodes_canvas->translate(offset_.x(), offset_.y());
-
-  if (context.raster_cache && display_list_raster_cache_item_) {
-    AutoCachePaint cache_paint(context);
-    if (display_list_raster_cache_item_->Draw(context,
-                                              cache_paint.sk_paint())) {
-      TRACE_EVENT_INSTANT0("flutter", "raster cache hit");
-      return;
-    }
-  }
+  auto save = context.state_stack.save();
```

Comment:
I know this `state_stack.save()` will return an `AutoRestore` Object but if I don't know the `save()` implement only read this line code I will misunderstand we forget to restore.

Reply:
Would a better name help?